lib/spack/spack/test/cmd/uninstall.py

```diff
@@ -70,14 +67,66 @@ def test_uninstall_spec_with_multiple_roots(
 
 
 @pytest.mark.db
-@pytest.mark.usefixtures('mutable_database')
 @pytest.mark.parametrize('constraint,expected_number_of_specs', [
     ('dyninst', 13), ('libelf', 13)
 ])
 def test_force_uninstall_spec_with_ref_count_not_zero(
-        constraint, expected_number_of_specs
+        constraint, expected_number_of_specs, mutable_database
 ):
     uninstall('-f', '-y', constraint)
 
     all_specs = spack.store.layout.all_specs()
     assert len(all_specs) == expected_number_of_specs
+
+
+@pytest.mark.db
+def test_force_uninstall(mutable_database):
```

Comment:
Does this test belong in `uninstall`?

Even if so, can you rename it something clearer? Maybe `test_force_uninstall_reinstall_by_hash`

Reply:
Renamed.